backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/database/Schema.kt

```diff
@@ -0,0 +1,24 @@
+package app.ehrenamtskarte.backend.migration.database
+
+import org.jetbrains.exposed.dao.IntEntity
+import org.jetbrains.exposed.dao.IntEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IdTable
+import org.jetbrains.exposed.sql.javatime.CurrentTimestamp
+import org.jetbrains.exposed.sql.javatime.timestamp
+
+object Migrations : IdTable<Int>() {
+    override val id = integer("version").entityId()
+    override val primaryKey = PrimaryKey(id)
```

Comment:
Not sure, but if you're fiddling with id and primaryKey, maybe you could instead just use a normal `Table`?
I think IdTable<Int> does not anything more than that:
https://github.com/JetBrains/Exposed/blob/master/exposed-core/src/main/kotlin/org/jetbrains/exposed/dao/id/IdTable.kt#L41